modules/nlu/src/backend/api.ts

```diff
@@ -209,11 +209,23 @@ export default async (bp: typeof sdk, state: NLUState) => {
     const { botId, id } = req.params
     const botEngine = state.nluByBot[botId].engine1 as ScopedEngine
     await botEngine.storage.deleteEntity(id)
-    scheduleSyncNLU(req.params.botId)
+    scheduleSyncNLU(botId)
 
     res.sendStatus(204)
   })
 
+  router.post('/train', async (req, res) => {
+    try {
+      const { botId } = req.params
+      const botEngine = state.nluByBot[botId].engine1 as ScopedEngine
+      scheduleSyncNLU(req.params.botId)
+      await botEngine.trainOrLoad(true)
+    } catch {
+      res.sendStatus(500)
+    }
+    res.sendStatus(200)
```

Comment:
Status 200 should be sent at the end of the try block, here it will send status twice when there is an error, causing an headers already sent error